lib/Dialect/LWE/IR/LWEOps.td

```diff
@@ -159,13 +165,21 @@ def LWE_RAddOp : LWE_BinOp<"radd", [SameOperandsAndResultRings, SameOperandsAndR
   }];
 }
 
+def LWE_RAddOp_Plain : LWE_CiphertextPlaintextOp<"radd_plain", [Commutative, SameOperandsAndResultPlaintextTypes, InferTypeOpAdaptor]> {
```

Comment:
```suggestion
def LWE_RAddPlainOp : LWE_CiphertextPlaintextOp<"radd_plain", [Commutative, SameOperandsAndResultPlaintextTypes, InferTypeOpAdaptor]> {
```

In HEIR we prefer a name like `LWE_RAddPlainOp` so the cpp op name will become `lwe::RAddPlainOp` instead of `lwe::RAddOp_Plain`. I do not think we have op name that has underscore in it. A mnemonic with underscore is accepted.

Reply:
Also, please add canonicalizer on ct-pt ordering to LWE_RAddPlainOp and LWE_RMulPlainOp, it is just copy-pasting what 948dd4f66 has done in BGVOps.td and BGVOps.cpp.